ci2/ci/build.py

```diff
@@ -686,20 +708,20 @@ def config(self, deploy):  # pylint: disable=unused-argument
 
 
 class CreateDatabaseStep(Step):
-    def __init__(self, code, deploy, name, deps, database_name, namespace):
-        super().__init__(name, deps)
+    def __init__(self, params, database_name, namespace):
+        super().__init__(json)
```

Comment:
I think this should be params

Reply:
Good catch, thanks, fixed.  I guess lint didn't catch it because we're importing the json package.